src/node_report.cc

```diff
@@ -505,26 +506,45 @@ static void PrintJavaScriptErrorProperties(JSONWriter* writer,
   writer->json_objectend();  // the end of 'errorProperties'
 }
 
+static void ErrorToString(Isolate* isolate,
+                          Local<Context> context,
+                          Local<Value> error,
+                          std::string* ss) {
+  if (error.IsEmpty()) {
+    return;
+  }
+
+  Local<Value> js_str;
+  // `ToString` is not available to Symbols.
+  if (error->IsSymbol() &&
+      error.As<v8::Symbol>()->ToDetailString(context).ToLocal(&js_str)) {
+    String::Utf8Value sv(isolate, js_str);
+    *ss = std::string(*sv, sv.length());
+  } else if (!error->IsObject() && error->ToString(context).ToLocal(&js_str)) {
+    String::Utf8Value sv(isolate, js_str);
+    *ss = std::string(*sv, sv.length());
+  } else if (error->IsObject() &&
+             error.As<Object>()
+                 ->Get(context, node::FIXED_ONE_BYTE_STRING(isolate, "stack"))
+                 .ToLocal(&js_str)) {
```

Comment:
I'd really like it if we could avoid introducing more code that silently discards exception... I know the node-report code is already not good at that, but maybe at least for this function we could forward empty `Maybe`s, so that it becomes easier to eventually fix this in the future?

Reply:
What do you expect by "forward" in this context?

Reply:
Returning an empty `v8::Maybe<bool>` value when encountering an empty `Maybe` or `MaybeLocal` value, and `Just(true)` otherwise

(or a `v8::Maybe<std::string>`, I think that would be fine here as well)